template/partial/settings.html

```diff
@@ -0,0 +1,49 @@
+<form class="ui form signup-form">
+	<div class="two fields">
+		<div class="field">
+			<label>Email</label>
+			<input type="text" name="email" id="email" placeholder="Email">
+			<div class="ui basic red right pointing prompt label transition hide"></div>
+		</div>
+		<div class="field">
+			<label>Name</label>
+			<input type="text" name="name" id="name" placeholder="Your complete name">
+			<div class="ui basic red left pointing prompt label transition hide"></div>
+		</div>
+	</div>
+	<div class="two fields">
```

Comment:
Is "two fields" a good name for div? Every element name should tell what the element's usage is?